Dockerfile.alpine

```diff
@@ -23,6 +23,8 @@ RUN  apk add --update --no-cache \
         php7-fileinfo \
         php7-simplexml \
         php7-session \
+        php7-dom \
+        php7-xmlwriter \
```

Comment:
I just confirmed; the current Alpine Dockerfile **does** break during composer install because of the lack of these two packages.